### ISSUE TITLE
Type 'number[]' is not assignable to type 'DateArray'

### ISSUE THREAD

**Issue by chrysb** [adamgibbons/ics#138]
Thanks for making this!

I'm unable to get the example code to work in TypeScript. 

```
const event = {
  start: [2018, 5, 30, 6, 30]
}
```

I also tried 
```
const start = moment(startsAt)
    .format('YYYY-M-D-H-m')
    .split('-')
```

(which btw returns an array of strings)

And seeing your type definition is `[number, number, number, number, number]`, I'm having trouble getting them to match up.

Why not just use an isoString in the first place for the start and end properties?

**Comment by robertpatan** [comment]
I have the same issue. 
I used: `start: [momentDate.year(), (momentDate.month() + 1), momentDate.date(), momentDate.hour(), momentDate.minute()]` . Even just puting `[1,2,3,4,5]` doesn't work.
The dts file seems to be maybe based on a older version of ts?
`Type 'number[]' is not assignable to type 'DateArray'`. Even after correcting this, you will get another ts error at status.
Anyway I ended up using 
`// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
  // @ts-ignore
`
